peekaboo/ruleset/rules.py

```diff
@@ -224,22 +225,42 @@ def evaluate(self, sample):
                            True)
 
 
-class OfficeMacroWithAutoActionRule(Rule):
+class OfficeMacroWithSuspiciousKeyword(Rule):
     """ A rule checking the sample for Office macros. """
-    rule_name = 'office_macro_with_auto_action'
+    rule_name = 'office_macro_with_suspicious_keyword'
+
+    def get_config(self):
+        # get list of keywords from config file
+        suspicious_keyword_list = self.get_config_value(
+            'keyword', [])
+        # convert into list of regexes
+        self.suspicious_keywords = []
+        # config.get_by_type does not support flags
+        # so we compile ourself
+        try:
+            for r in suspicious_keyword_list:
+                self.suspicious_keywords.append(re.compile('.*'+r+'.*', re.IGNORECASE))
```

Comment:
What's the rationale for '.*' here? I'd rather leave that up to the user and use `re.match()` or just switch to `re.search()` instead. I'd opt for `re.search()` as in `CuckooEvilSigRule` because that way the user can still add '^' and '$' to anchor it which the above doesn't allow.